howto/ipaddress.po

```diff
@@ -87,24 +103,33 @@ msgid ""
 "use the :func:`ipaddress.ip_address` factory function, which automatically "
 "determines whether to create an IPv4 or IPv6 address based on the passed in "
 "value:"
-msgstr ""
+msgstr "Les adresses, souvent dénommées \"adresses hôtes \" sont les unités "
+"les plus basiques quand on travaille avec l'adressage IP. "
+"Le moyen le plus simple de créer des adresses est d'utiliser la fonction de "
+"fabrique :func:`ipaddress.ip_address` qui va automatiquement déterminer quoi "
+"créer entre une adresse IPv4 ou une adresse IPv6 en fonction de la valeur qui "
+"est transmise:"
 
 #: ../Doc/howto/ipaddress.rst:61
 msgid ""
 "Addresses can also be created directly from integers. Values that will fit "
 "within 32 bits are assumed to be IPv4 addresses::"
-msgstr ""
+msgstr "Les adresses peuvent être créées directement depuis des entiers. "
+"Les valeurs qui correspondent à des entiers 32 bits sont assimilées à des "
+"adresses IPv4::"
```

Comment:
Le `::` se traduit ` : ::` (c.f. https://github.com/python/python-docs-fr/#what-to-translate) pour conserver l'espace avant le ":" tout en conservant le marqueur d'introduction de bloc.

Reply:
Non `<insécable>:<insécable>::` c'est juste se compliquer la vie.
Soit tu mets `<espace>:<espace>::` mais c'est pas super parce qu'avant un `:` normalement il faut un insécalbe.
Soit tu mets `<insécalbe>::`.